src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ReaderMetricsProcessorTests.java

```diff
@@ -222,11 +223,18 @@ public void testAlignNodeMetrics() throws Exception {
             masterPendingFinal);
 
     Result<Record> res = alignedWindow.fetchAll();
-    assertTrue(1 == res.size());
+        assertTrue(2 == res.size());
     Field<Double> valueField =
         DSL.field(MasterPendingValue.MASTER_PENDING_QUEUE_SIZE.toString(), Double.class);
+        Field<String> dimensionField =
+                DSL.field(MasterPendingTaskDimension.MASTER_PENDING_TASK_TYPE.toString(), String.class);
     Double pending = Double.parseDouble(res.get(0).get(valueField).toString());
-    assertEquals(2.2d, pending, 0.001);
+        assertEquals(1.0d, pending, 0.001);
```

Comment:
why this change from 2.2d to 1.0d?

Reply:
Earlier we have only PendingQueue size and we have 3 windows in test case. It will calculate the three window values and result was 2.2. Now we added tasktype so table looks like
```
-------------------------------------------------------
|Master_PendingTaskType | Master_PendingQueueSize |
-------------------------------------------------------
| create-index          | 1.0                     |
| updateSnapshot       | 3.0                      |
--------------------------------------------------------
```